proposals/bring_your_own_argo_workflows/byoaw_test_plan.md

```diff
@@ -0,0 +1,631 @@
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Test Case Summary** | Verify run of "Pipelines with both input and output artifacts" pipeline                                                                                                                  |
+| **Test Steps**        | <ol><li> Configure DSPA with external Argo </li><li> Execute Pipeline - Pipelines with both input and output artifacts </li><li> Verify each pipeline type executes correctly</li></ol>  |
+| **Expected Results**  | - Pipeline runs successfully<br/>- Upstream and Downstream components can produce & consume artifacts                                                                                    |
+
+| Test Case ID          | TC-PF-011                                                                                                                                                                    |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Test Case Summary** | Verify run of "Pipelines without input parameters" pipeline                                                                                                                  |
+| **Test Steps**        | <ol><li> Configure DSPA with external Argo </li><li> Execute Pipeline - Pipelines without input parameters </li><li> Verify each pipeline type executes correctly</li></ol>  |
+| **Expected Results**  | - Pipeline runs successfully                                                                                                                                                 |
+
+| Test Case ID          | TC-PF-012                                                                                                                                                                 |
+|-----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Test Case Summary** | Verify run of "Pipelines with NO output artifacts, but just input artifacts" pipeline                                                                                     |
+| **Test Steps**        | <ol><li> Configure DSPA with external Argo </li><li> Execute Pipeline - Pipelines with output artifacts </li><li> Verify each pipeline type executes correctly</li></ol>  |
+| **Expected Results**  | - Pipeline runs successfully<br/>- Output artifacts (like a model/trained data) are produced to S3 correctly                                                              |
```

Comment:
The summary doesn't seem to match the steps and expected results.
`Verify run of "Pipelines with NO output artifacts`
`Execute Pipeline - Pipelines with output artifacts`
`Output artifacts (like a model/trained data) are produced to S3 correctly`